spacetemplate/assets/base.yaml

```diff
@@ -158,22 +158,22 @@ work_item_link_types:
   name: Blocker
   description: One work item item blocks another one
   forward_name: blocks
-  forward_description: Point to a work item that is blocked by this one.
+  forward_description: Link the work item that is blocked by this one.
```

Comment:
@Preeticp We shouldn't use the word "link" here because it doesn't involve a **direction**. The "point to" was better IMHO because it contained this direction. And the whole reason for having these descriptions is to make this more meaningful.

Reply:
In technical writing, 'point to' is used to ask users to move the mouse pointer over a screen element. Since this is a drop-down the user action is to Select the option from the drop-down. 
I would suggest 'Select the' instead of 'Point/Link to', however please note, in this drop-down the user is actually selecting the relationship/link type and not selecting the WI/pointing to it. So 'Point to /link to /Select the work item that blocks this one.' is an action that applies to the entire sequence of actions, i.e user select the type of link/relationship and then the WI. So having these descriptions in the link type drop-down is slightly misleading.

Also, the descriptions in the drop-down is seen by users only if they click the drop-down, information which imho should be upfront is hidden in the drop-down.

Here is a suggestion: Beneath the heading 'Create Link', add a line telling the user what he is expected to do. This would be upfront and imho of greater help. 
Something like: Select a work item to link this work item to. / Create links to other work items and establish relationships between them. / Link this work item to other work items and create relationships between them.
Other issue trackers I compared to also seem to follow a similar pattern.

Thoughts?

Reply:
I'd prefer the `Select the` solution for now. Can you make the changes please?